### PR TITLE
xsoar branding added

### PR DESCRIPTION
**Pull request by csestito**
## Status
Ready w/ approval from @fvigo and @mchasepan 

## Related Issues
updating branding

## Description
Updated look and feel of website to match new branding

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -87,10 +110,10 @@ module.exports = {
         {
           title: "Docs",
           items: [
-            { to: "docs/howtos/integrations/getting-started-guide", label: "Development Guide" },
-            { to: "docs/partners/why-demisto", label: "Why become a Partner?" },
+            { to: "/docs/integrations/getting-started-guide", label: "Development Guide" },
```
Comment (fvigo) [inline docusaurus.config.js:113]: Please point this to /docs/welcome and label "Developer Docs"

File: src/pages/index.js [hunk 2]
```diff
@@ -141,32 +141,32 @@ function Home() {
...
               <div className={styles.indexCtas}>
                 <Link
                   className={classnames(
-                    "button button--outline button--secondary button--lg",
-                    styles.indexCtasGetStartedButton
+                    "button button--info button--secondary button--lg",
+                    styles.headerButtons
                   )}
-                  to="docs/why-demisto"
+                  to="docs/partners/why-xsoar"
```
Comment (fvigo) [inline src/pages/index.js:162]: To be consistent with the rest please point it to `/docs/partners/why-xsoar` with the heading `/`

File: src/pages/index.js [hunk 3]
```diff
@@ -193,10 +193,10 @@ function Home() {
         <section className={styles.hero}>
           <div className="customer">
             <div className="customer-inner">
-              Are you a Demisto Customer or an Independent Developer?
-              <Link to="docs/getting-started-guide"> Get Started Now!</Link>
+              Are you a Cortex XSOAR Customer or an Independent Developer?
+              <Link to="docs/integrations/getting-started-guide"> Get Started Now!</Link>
```
Comment (fvigo) [inline src/pages/index.js:197]: Let's point them to the Docs Welcome page: `/docs/welcome`

**Comment by fvigo** [comment]
@mchasepan  please review and approve now that the build works

**Comment by mchasepan** [comment]
@fvigo thx, how can I see the preview? I dont have it built locally, but the version @csestito showed me looked great.

**Comment by fvigo** [comment]
@mchasepan  click on the `Details` link under `deploy/netlify`

image

**Comment by fvigo** [comment]
Resolved footer conflict

**Pull request merged**